nwb_conversion_tools/nwbconverter.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, **source_data):
         self.data_interface_objects = {
             name: data_interface(**source_data[name])
             for name, data_interface in self.data_interface_classes.items()
+            if name in source_data
```

Comment:
Adding this allows a given Converter class to contain an optional data interface class (like a recording) that only gets instantiated if we also include it in the keys of the `source_data` input arguments (which is easier to control than adding `pop`-like functionality to the `init` of that converter).